tests/test_photo.py

```diff
@@ -174,9 +174,9 @@ def test_send_bytesio_jpg_file(self, bot, chat_id):
         assert isinstance(photo.file_id, str)
         assert photo.file_id != ''
         assert isinstance(photo, PhotoSize)
-        assert photo.width == 1920
-        assert photo.height == 1080
-        assert photo.file_size == 30907
+        assert photo.width == 1280
```

Comment:
What changed here?
And how... like it's a smaller image but it takes up more space? :/

Reply:
I have no Idea, but it's what came back...